osquery/tables/applications/darwin/browser_plugins.cpp

```diff
@@ -66,6 +92,30 @@ const std::map<std::string, std::string> kSafariExtensionKeys = {
     {"DeveloperIdentifier", "developer_id"},
 };
 
+struct SandboxedExtensionData {
+  /// Extension Name
+  std::string name;
+
+  /// Extension Identifier
+  std::string identifier;
+
+  /// Extension Version
+  std::string version;
+
+  /// Extension SDK Version
+  std::string sdk;
+
+  /// Parent App
+  std::string app_path;
+};
+
+using SandboxedExtensionsData = std::vector<SandboxedExtensionData>;
+
+const std::vector<std::string> kExtensionsAppDirsToExclude = {
+    "/Xcode.app",
+    "/Safari.app",
+};
```

Comment:
Seems like maybe you are excluding these because they are built in extensions? I could see it perhaps being useful to include them anyway.

Reply:
Yes, I wasn't sure to include built-in extensions. I've just removed this logic to include them in the query results.

Reply:
Removing feels weird. I'd leave them.

Especially because we're trawling disk for plists, we don't actually know that whatever is in a directory name Xcode.app is actually xcode. Etc.

Reply:
Than for the comment. I've just added this logic back to the ced